app/domain/authentication/auth_data.rb

```diff
@@ -0,0 +1,15 @@
+
+module AuthData
```

Comment:
I think it's fine to leave this as a module, but as a note, this is actually a great candidate for inheritance.